.github/labeler.yml

```diff
@@ -0,0 +1,69 @@
+---
+# Each entry in this file is a label that will be applied to pull requests
+# if there is a match based on the matching rules for the entry. Please see
+# the actions/labeler documentation for more information:
+# https://github.com/actions/labeler#match-object
+#
+# Note: Verify that the label you want to use is defined in the
+# crazy-max/ghaction-github-labeler configuration file located at
+# .github/labels.yml.
+
+# Enable if Ansible playbooks are used in the repository.
+ansible:
+  - changed-files:
+      - any-glob-to-any-file:
+          - defaults/**/*.yml
+          - handlers/**/*.yml
+          - meta/**/*.yml
+          - tasks/**/*.yml
+          - vars/**/*.yml
+dependencies:
+  - changed-files:
+      - any-glob-to-any-file:
+          # Add any dependency files used.
+          - .pre-commit-config.yaml
+          - requirements*.txt
```

Comment:
Do we want to use either of the files in `meta/` to trigger this label?

Reply:
Please see commit c9d93a9.